### ISSUE TITLE
BUG: Most ITK Python type hints do not show up in VS Code (Windows)

### ISSUE THREAD

**Issue by tbirdso** [InsightSoftwareConsortium/ITK#3486]
### Description

[https://github.com/InsightSoftwareConsortium/ITK/pull/3444](https://github.com/InsightSoftwareConsortium/ITK/pull/3444) and its predecessors such as [https://github.com/InsightSoftwareConsortium/ITK/pull/3200](https://github.com/InsightSoftwareConsortium/ITK/pull/3200) were intended to introduce/improve type hints with the ITK Python package; however, only a subset of Pythonic type hints appear in VS Code using a pip-installed ITK package.

### Steps to Reproduce

1. `pip install itk`
2. Install VS Code with Python extensions, set Python interpreter
3. Create a new Python file
```py
import itk
itk.
```
4. View list of suggestions after `itk.` in line 2

### Expected behavior

Most/all values in `itk` namespace are viewable in list. For instance, `itk.Image`, `itk.join_series_image_filter`

### Actual behavior

Only functions/types defined in [https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itk/support/extras.py](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itk/support/extras.py) or [https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itk/support/types.py](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itk/support/types.py) appear to be represented in autocomplete suggestions

### Versions

ITK v5.3rc4

### Environment

Windows 10
VS Code 1.62

**Comment by PranjalSahu** [comment]
@tbirdso yes, I mostly see items from extras.py like you mentioned. And I didn't see itk.Image.
![Screenshot from 2022-07-14 16-17-21](https://user-images.githubusercontent.com/1044135/179075977-a3bc5c7c-f023-4e00-927c-4188b554494a.png)

**Comment by tbirdso** [comment]
Thanks for verifying @PranjalSahu . Perhaps @hjmjohnson or @kian-weimer could weigh in on whether this issue was observed/anticipated from their testing surrounding #3200?

cc @thewtex

**Comment by thewtex** [comment]
Have you tried `5.3rc4.post1`?

**Comment by tbirdso** [comment]
Hi @thewtex , good point, I just updated to `5.3rc04.post1` for my environment referenced by VS Code and am seeing the same behavior.
